packaging/linux/systemd/keybase-redirector.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Keybase Root Redirector for KBFS
+Wants=keybase.service
+
+[Service]
+Type=simple
+
+# Abort if root redirector is explicitly disabled
+ExecStartPre=/usr/bin/keybase -c /etc/keybase/config.json config get --direct --assert-false --assert-ok-on-nil disable-root-redirector
```

Comment:
This looks cool! Didn't seem to work on macOS when I played with it though. Does it give an exit code 0 only if assert passes, i.e. `disable-root-redirector==false`?

Reply:
Got it; thanks for the explanation. That's neat! Both names sound OK to me as long as it's explained in the CLI doc.